plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsFlywayMigration.java

```diff
@@ -24,26 +26,24 @@
 import static org.testng.Assert.assertEquals;
 
 @Test(singleThreaded = true)
```

Comment:
Reminder to ensure that all subclasses include `Test(singleThreaded = true)` too since if the subclass doesn't have any `@Test` method then the single-threaded is lost. See `forceTestNgToRespectSingleThreaded` methods in existing code.

Reply:
I see that it's added to Oracle tests in next commit.